parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -253,7 +253,10 @@ installImg() {
 
 extractHyperkube() {
     CLI_TOOL=$1
-    path="/home/hyperkube-downloads/${KUBERNETES_VERSION}"
+    # CUSTOM_CLOUD_SUFFIX only set when extracting custom Hyperkube builds (e.x.: Azure Stack)
+    CUSTOM_CLOUD_SUFFIX=$2
+    kubernetes_version="${KUBERNETES_VERSION}${CUSTOM_CLOUD_SUFFIX}"
```

Comment:
use a different var name here to avoid confusion with `KUBERNETES_VERSION`

Reply:
on second thought, wouldn't it be cleaner to not touch this function and add the `AZURE_STACK_SUFFIX` to the `KUBERNETES_VERSION` var directly in install_dependencies.sh?

Reply:
Sorry, I pushed changes before seeing these comments. Yes, no problem.